cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -243,6 +243,7 @@ def test_revert_active_snapshot(full_test_vm, provider, soft_assert, register_ev
                            active_snapshot=True)
 
 
+@pytest.mark.rhv3
```

Comment:
I think this should be tier 2

Reply:
@istein1 Actually here I have to disagree. If you check the description, it just checks one button for being clickable. I cannot imagine such test anywhere out of tier3.

Reply:
After discussion on what this test do offline, I agree it should be tier 3.